tests/test_serde.rs

```diff
@@ -297,18 +297,30 @@ fn test_strings_needing_quote() {
     #[derive(Serialize, Deserialize, PartialEq, Debug)]
     struct Struct {
         boolean: String,
+        yes_scalar: String,
```

Comment:
Ugly names so they don't get quoted. But should really be fixed and then called just `yes`, `no`, etc.